contracts/stats/StatBlock.sol

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+/**
+ * Authors: Moonstream Engineering (engineering - at - moonstream.to)
+ * GitHub: https://github.com/moonstream-to/web3
+ */
+
+import {IERC1155} from "@openzeppelin-contracts/contracts/token/ERC1155/IERC1155.sol";
+import {IStatBlock} from "./IStatBlock.sol";
+
+contract StatBlock is IStatBlock {
+    address AdminTerminusAddress;
+    uint256 AdminTerminusPoolID;
+    // Stats are 0-indexed.
```

Comment:
I use 1-indexing for items that could be values on a mapping because 0 is the default value and I don't want to confuse them.

Although there is no such mapping on the `StatBlock` itself, such a mapping could easily exist on a contract consuming `StatBlock`. So I agree with you, I will change stats to be 1-indexed.